### PR TITLE
Added delete post functionality

### LINKED ISSUE #117: Option to delete Post in frontend
API : https://sociomark-backend.herokuapp.com/post/delete

### PR DESCRIPTION
**Pull request by DebugAgrawal**
> ### Now user can delete their own post 🚩

 Resolves #117  

image

### CODE REVIEW

**Comment by aitikgupta** [comment]
Can we do a X icon instead of "DELETE"? @bodhisha your css skillz needed. XD

**Review by bodhisha — COMMENTED** [review]
> Can we do a X icon instead of "DELETE"? @bodhisha your css skillz needed. XD

Yup yup. I think a small 🗑️ icon in red color would also look good with the UI? Text seems bit large with other icons

**Comment by DebugAgrawal** [comment]
> > Can we do a X icon instead of "DELETE"? @bodhisha your css skillz needed. XD
> 
> Yup yup. I think a small 🗑️ icon in red color would also look good with the UI? Text seems bit large with other icons

🗑️ icon sounds good

image

done 👍🚩

File: frontend/src/constants.js [hunk 1]
```diff
@@ -6,14 +6,15 @@ switch (ENV) {
     BASE_URL = "https://sociomark-backend.herokuapp.com";
     break;
   default:
-    BASE_URL = "http://localhost:8000";
+    BASE_URL = "https://sociomark-backend.herokuapp.com";
```
Comment (bodhisha) [inline frontend/src/constants.js]: By mistake right? @DebugAgrawal

**Review by bodhisha — CHANGES_REQUESTED** [review]
Looks good! 🎉 Just a suggestion to change the URL back to `http://localhost:8000`

**Review by aitikgupta — APPROVED** [review]
Awesomeee! 🎉

**Review by bodhisha — APPROVED** [review]

**Pull request merged**